Bomb.py

```diff
@@ -0,0 +1,39 @@
+import pygame
+
+class Bomb:
+    def __init__(self, x, y, power):
+        self.x = x
+        self.y = y
+        self.power = power
+        self.timer = 0
+
+    def __del__(self):
+        pass
+
+    def plant(self, win):
+        file = 0
+        if self.timer <= 40:
+            file = 'assets/bomb/bomb1.png'
+        elif self.timer <= 80:
+            file = 'assets/bomb/bomb2.png'
+        elif self.timer <= 120:
+            file = 'assets/bomb/bomb3.png'
+        elif self.timer <= 160:
+            file = 'assets/bomb/fire.png'
```

Comment:
We shouldn't hardcode these values - 40, 80.. We also made it in Player.py so we need to change them to constants

You can also make constants with file paths to these images